src/MOIWrapper.jl

```diff
@@ -0,0 +1,283 @@
+export CplexOptimizer
+
+using LinQuadOptInterface
+const LQOI = LinQuadOptInterface
+const MOI  = LQOI.MOI
+
+const SUPPORTED_OBJECTIVES = [
+    LQOI.Linear
+]
+
+const SUPPORTED_CONSTRAINTS = [
+    (LQOI.Linear, LQOI.EQ),
+    (LQOI.Linear, LQOI.LE),
+    (LQOI.Linear, LQOI.GE),
+    (LQOI.SinVar, LQOI.EQ),
+    (LQOI.SinVar, LQOI.LE),
+    (LQOI.SinVar, LQOI.GE),
+    (LQOI.SinVar, LQOI.IV),
+    (LQOI.SinVar, MOI.ZeroOne),
+    (LQOI.SinVar, MOI.Integer),
+    (LQOI.VecVar, MOI.Nonnegatives),
+    (LQOI.VecVar, MOI.Nonpositives),
+    (LQOI.VecVar, MOI.Zeros),
+    (LQOI.VecLin, MOI.Nonnegatives),
+    (LQOI.VecLin, MOI.Nonpositives),
+    (LQOI.VecLin, MOI.Zeros)
+]
+
+mutable struct CplexOptimizer <: LQOI.LinQuadOptimizer
+    LQOI.@LinQuadOptimizerBase    
+    env::Env # Cplex environment
+    CplexOptimizer(::Void) = new()
+end
+
+function LQOI.LinearQuadraticModel(::Type{CplexOptimizer},env)
+    env = Env()
+    return Model(env::Env)
+end
+
+function CplexOptimizer(;kwargs...)        
+    env = Env()
+    for (name,value) in kwargs
+        set_param!(env, string(name), value)
+    end
+    m = CplexOptimizer(nothing)
+    m.env = env
+    return m
+end
+
+LQOI.supported_constraints(s::CplexOptimizer) = SUPPORTED_CONSTRAINTS
+LQOI.supported_objectives(s::CplexOptimizer)  = SUPPORTED_OBJECTIVES
+
+LQOI.backend_type(m::CplexOptimizer, ::MOI.EqualTo{Float64})     = Cchar('E')
+LQOI.backend_type(m::CplexOptimizer, ::MOI.LessThan{Float64})    = Cchar('L')
+LQOI.backend_type(m::CplexOptimizer, ::MOI.GreaterThan{Float64}) = Cchar('G')
+
+LQOI.backend_type(m::CplexOptimizer, ::MOI.Zeros)                = Cchar('B')
+LQOI.backend_type(m::CplexOptimizer, ::MOI.Nonpositives)         = Cchar('L')
+LQOI.backend_type(m::CplexOptimizer, ::MOI.Nonnegatives)         = Cchar('E')
+
+# TODO - improve single type
+function LQOI.change_variable_bounds!(instance::CplexOptimizer, 
+        colvec::Vector{Int}, valvec::Vector{Float64}, sensevec::Vector{Cchar})
```

Comment:
Style: use English understandable names. `columns` instead of `colvec` and so on.